rust/snippets/spice-rs/main.rs

```diff
@@ -1,25 +1,22 @@
+use spice_rs::Client;
 use futures::stream::StreamExt;
-use spice_rs::*;
 
-let mut spice_client = new_spice_client(api_key).await;
+let mut spice_client = Client::new("api_key").await;
 
 match spice_client.query(
-    "SELECT number, \"timestamp\", base_fee_per_gas, base_fee_per_gas / 1e9 AS base_fee_per_gas_gwei FROM eth.recent_blocks limit 10".to_string()
+    "SELECT number, \"timestamp\", base_fee_per_gas, base_fee_per_gas / 1e9 AS base_fee_per_gas_gwei FROM eth.recent_blocks limit 10"
     ).await {
         Ok(mut flight_data_stream) => {
-              // Read back RecordBatches
+            // Read back RecordBatches
             while let Some(batch) = flight_data_stream.next().await {
             match batch {
-                Ok(batch) => {
-                    println!("{:?}", batch);
-                },
-                Err(e) => {
-                    assert!(false, "Error: {}", e)
+                Ok(batch) => {                   
+                    /* process batch */
+                    println!("{:?}", batch)
                 },
+                Err(e) => { /* handle error */ },
             };
             }
         }
-        Err(e) => {
-            assert!(false, "Error: {}", e);
-        }
+        Err(e) => { /* handle error */ }
```

Comment:
Can we add `assert!(false, "Error: {}", e);` back. we want users to be able to copy/paste the exact snippet and have it work (or in this case, knowingly fail), even if they don't know the language super well.